### PR TITLE
Feature/fix import files command

### PR DESCRIPTION
**Pull request by jdcookie**
Update the import_files command so that is does not use deprecated methods and classes.
Works with Django 1.10+
A simple edit to resolve Issue #966

### CODE REVIEW

**Comment by skirsdeda** [comment]
@yakky So maybe I should revive #500? It would also make importfiles management command more useful.

**Comment by yakky** [comment]
@skirsdeda my plan is actually to review both
I would like to keep 1.3.1 "easy", and thus including this to make the command at least work across all the django version.
I think that #500 is more suitable for release 1.4, which I think we could release shortly (just dropping older django and including the PR already approved / evaluated for a shorter release cycle)

**Comment by skirsdeda** [comment]
@yakky sounds good.

File: filer/management/commands/import_files.py [hunk 1]
```diff
@@ -81,7 +80,7 @@ def walker(self, path=None, base_folder=None):
         This method walk a directory structure and create the
         Folders and Files as they appear.
         """
-        path = path or self.path
+        path = path or self.path if self.path is not False else ''
```
Comment (yakky) [inline filer/management/commands/import_files.py]: path must be a string in any case, I think we should rewrite this as `path or self.path or ''`

Comment (jdcookie) [reply filer/management/commands/import_files.py]: ok - will update it now.

File: filer/management/commands/import_files.py [hunk 2]
```diff
@@ -108,27 +107,30 @@ def walker(self, path=None, base_folder=None):
...
     """
-
-    option_list = BaseCommand.option_list + (
-        make_option('--path',
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
             action='store',
             dest='path',
             default=False,
```
Comment (yakky) [inline filer/management/commands/import_files.py]: Can I ask you to mark path as required? This https://github.com/divio/django-filer/pull/1058/files#diff-361fe6a032f5ff57a38ea06ab3c367a0R83 fixes an issue if path is empty, but it opens the door to an undesired behavior (importing the whole django project root)

Comment (jdcookie) [reply filer/management/commands/import_files.py]: Should we drop the default then, if its required no need to have a default value?

Comment (yakky) [reply filer/management/commands/import_files.py]: Yes, we can remove it, as it's redundant

**Comment by jdcookie** [comment]
Should be working now
```
root@9e22bc1e495a:/usr/src/app# ./manage.py import_files
usage: manage.py import_files [-h] [--version] [-v {0,1,2,3}]
                              [--settings SETTINGS] [--pythonpath PYTHONPATH]
                              [--traceback] [--no-color] --path PATH
                              [--folder BASE_FOLDER]
manage.py import_files: error: the following arguments are required: --path
```

**Comment by yakky** [comment]
Thanks @jdcookie !
Waiting for tests to finish to merge

**Comment by yakky** [comment]
Merged, thanks again for working on this

**Comment by jdcookie** [comment]
No worries

**Pull request merged**